source/monitoring.txt

```diff
@@ -35,6 +35,104 @@ To learn more about logging in the connector, see the
 
 .. _kafka-monitoring-information:
 
+Use Cases
+---------
+
+This section describes use cases for monitoring MongoDB source and sink
+connectors.
```

Comment:
It seems that users would have questions of the forms:

1. How much in total/at present?
2. How much over a span of time?
3. Is the rate changing?
4. What is the value compared to some other metric?

It might be worth explicitly pointing out that we track only the first, and the user is expected to compute or visually determine the remaining. I'm not sure whether to suggest enumerating these different types as I have done above, but the user might be surprised that they cannot immediately see metrics like "rate over the last minute" or "% of time spent in phase X", and are expected to compute them in some way (it wasn't immediately obvious to me when implementing this that we would not be offering such metrics).

Examples:

1. Have I been receiving all records?
2. Is there a high constant rate of errors? How many records, histogram by hours of the day?
3. Is the error rate increasing? Are we falling further and further behind the origin?
4. Is the problem in MongoDB or Kafka, and if in MongoDB, which phase is causing a problem?

(I don't suggest that we do or do not add these examples, they are meant to be illustrative.)

Reply:
Sounds good, agree it could be useful to explicitly state what types of metrics the connector provides (counts and most recent occurrence) and when you must compute something (anything that cannot be expressed as a count or most recent occurrence).

Added a subsection explaining this and link to it from use cases subsection.